app/experimenter/docs/adrs/0005-doc-hub.md

```diff
@@ -0,0 +1,225 @@
+# Use [TBD] for the Nimbus User Doc Hub
+
+- Status: proposed
+- Deciders: Lauren Zugai, Kate Hudson, Jody Heavener, [TBD]
+- Date: 2021-02-16
+
+Technical Story: [Documentation Hub for Nimbus Users](https://jira.mozilla.com/browse/EXP-860)
+
+## Context and Problem Statement
+
+In scaling Nimbus for 2021 and beyond, we need to provide experiment owners, data scientists, and desktop and Fenix engineers documentation to better streamline working with the platform. Following the same line of thought in [`nimbus-shared` ADR 0001](https://github.com/mozilla/nimbus-shared/blob/main/docs/adr/0001-one-console-app.md), users should expect a single source of truth for their documentation needs.
+
+The documentation that will be part of the hub includes:
```

Comment:
@k88hudson Is this accurate, or are there other docs that should be included on the list?

Reply:
In terms of use-cases:
* A front page 
* SDK docs for integration in Fenix (not yet written to my knowledge)
 * SDK docs for integration in Desktop Front end (https://www.notion.so/k88hudson/Nimbus-Migration-Guide-for-Desktop-Front-End-Experiments-d36c21e505f84550aad1202897fc4ba3)
 * Docs about how to use the console (from the perspective of a person launching an experiment. This could live in experimenter or link out to mana)
 * Docs for how to read automated analysis (again, could live here or  link out to mana)
 * Docs for how to add jetstream configs, for data scientists.
 * A bunch of other google docs and mana links.

In the future, we will also need to write iOS and gecko integration docs, but we don't need to now since they don't exist.

Reply:
@k88hudson perf, will update this section with my next push👌